src/components/widgets/Accordion.js

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useRef, useState } from 'react';
+import hljs from 'highlight.js/lib/core';
+import 'highlight.js/styles/night-owl.css';
+import html from 'highlight.js/lib/languages/xml';
+
+function AccordionItem({ title, children }) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleAccordionTitleClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <div className="accordion-item">
+      <h3 className={`accordion-title ${isOpen ? 'arrow-up' : 'arrow-down'}`} onClick={handleAccordionTitleClick}>{title}</h3>
+      {isOpen && (
+        <div className="accordion-content">
+          {children}
+        </div>
+      )}
+    </div>
+  );
+}
+
+export default function Accordion() {
+  const codeRef = useRef(null);
+  const [buttonText, setButtonText] = useState('Copy code');
+
+  hljs.registerLanguage('html', html);
+  useEffect(() => {
+    hljs.highlightAll();
+  }, []);
+
+  const handleCopyCode = () => {
+    const codeElement = codeRef.current;
+    const range = document.createRange();
+    range.selectNode(codeElement);
+    window.getSelection().removeAllRanges();
+    window.getSelection().addRange(range);
+    document.execCommand('copy');
+    window.getSelection().removeAllRanges();
+
+    setButtonText('Copied!');
+
+    setTimeout(() => {
+      setButtonText('Copy code');
+    }, 2000);
+  };
+
+  return (
+    <section className="wd-content" id="toc-accordion">
+      <h2 id="accordion" className="section-top">Accordion</h2>
+      <p>Use the <strong>Accordion</strong> to display content in collapsible compartments.</p>
+      <p>Use <span className="wd-monospace">&lt;div className="accordion"&gt;&lt;/div&gt;</span> to wrap your content.</p>
```

Comment:
Change `className` to `class`

Reply:
I think this should stay `className` since it's not part of the template literal section.

Reply:
Just on line 54 it should change because it is within the <p> tag so it shows us as text.

Reply:
Ah gotcha, I thought you meant the `className="wd-monospace"` part.